index.js

```diff
@@ -1,27 +1,43 @@
 /* globals require, __dirname, process */
 
 const path = require('path');
+const builder = require('adguard-hostlists-builder');
 const fs = require('fs');
-const builder = require("adguard-hostlists-builder");
+const { checkRemovedServices } = require('./scripts/services/check-removed-services');
+const { rewriteServicesJSON } = require('./scripts/services/rewrite-services-json');
+
+const filtersDir = path.join(__dirname, './filters');
+const assetsDir = path.join(__dirname, './assets');
+const tagsDir = path.join(__dirname, './tags');
+const localesDir = path.join(__dirname, './locales');
+const servicesDir = path.join(__dirname, './services');
+const servicesJSON = path.join(assetsDir, '/services.json');
+
+// build services.json from services folder
+const buildServices = async (dir, JSON) => {
```

Comment:
1. jsdoc comments are needed here as well

2. param names should be better, e.g. dirPath instead of dir, filePath or fileName instead of JSON

3. when the function is called
   `await buildServices(servicesDir, servicesJSON);`
   it is actually
    ```
    await buildServices(
        path.join(__dirname, './services'),
        path.join(assetsDir, '/services.json'),
    );
    ```
    and because of `assetsDir` it becomes
    ```
    await buildServices(
        path.join(__dirname, './services'),
        path.join(path.join(__dirname, './assets'), '/services.json'),
    );
    ```
    that's too much of path.join() calls, and just does not look right

**after names of function parameters are changed and jsdoc comments are added, it will be much easier to detect such issues**